google/cloud/spanner/client.cc

```diff
@@ -49,16 +49,18 @@ StatusOr<ResultSet> Client::Read(Transaction transaction, std::string table,
                       std::move(columns), std::move(read_options)});
 }
 
-StatusOr<ResultSet> Client::Read(SqlPartition const& /*partition*/) {
-  return Status(StatusCode::kUnimplemented, "not implemented");
-}
-
-StatusOr<std::vector<SqlPartition>> Client::PartitionRead(
-    Transaction const& /*transaction*/, std::string const& /*table*/,
-    KeySet const& /*keys*/, std::vector<std::string> const& /*columns*/,
-    ReadOptions const& /*read_options*/,
-    PartitionOptions const& /*partition_options*/) {
-  return Status(StatusCode::kUnimplemented, "not implemented");
+StatusOr<ResultSet> Client::Read(ReadPartition const& partition /*partition*/) {
+  return conn_->Read(partition);
+}
+
+StatusOr<std::vector<ReadPartition>> Client::PartitionRead(
+    Transaction const& transaction /*transaction*/,
+    std::string const& table /*table*/, KeySet const& keys /*keys*/,
```

Comment:
You can remove the `/*table*/` argument comments, which are unneeded now that we have actual argument names.

Reply:
Done.